internal/configs/node/network.go

```diff
@@ -80,7 +80,7 @@ const (
 	// DefaultWaitForEachPeerToConnect sets the sync configs to connect to neighbor peers one by one and waits for each peer to connect.
 	DefaultWaitForEachPeerToConnect = false
 	// DefaultNoTransportSecurity
-	DefaultNoTransportSecurity = false
+	DefaultNoTransportSecurity = true
```

Comment:
and here as well. Let's make sure it is backward compatible with current main-net validators configs.

Reply:
same comment as above